app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -910,7 +910,7 @@ export default Ember.Controller.extend({
       opts.topicTitle &&
       opts.topicTitle.length <= this.siteSettings.max_topic_title_length
     ) {
-      this.set("model.title", escapeExpression(opts.topicTitle));
```

Comment:
Any chances this could introduce an XSS? I think this was added in c28c5083e0e7db96ec27f27c0107292e7469b214

Reply:
This is what I also wrote in the top. 

"But similar to what tgxworld commented under the commit, I was also not able to inject html or js into the title using the url method. Tried a lot of variations and from my point of view its safe to remove the escape function."

Reply:
My bad, went directly to the "Files changed" tab...